push/src/main/java/org/aerogear/mobile/push/UnifiedPushConfig.java

```diff
@@ -22,7 +29,29 @@ public void setAlias(String alias) {
     }
 
     public void setCategories(List<String> categories) {
+        validateCategories(categories);
         this.categories = new ArrayList<>(categories);
```

Comment:
a nonNull check should be added to avoid NullPointerException here

Reply:
Done!